### ISSUE TITLE
Unexpected token: operator (>) / export 'default' was not found in 'vue-multiselect'

### ISSUE THREAD

**Issue by Namesonic** [monterail/vue-multiselect#506]
I have a vue-cli webpack template and am getting a production build error that I can't solve.  I've tried with the following versions of vue-multiselect

vue-multiselect@2.0.0-beta.15  `npm install vue-multiselect@next`
vue-multiselect@2.0.2 `npm install vue-multiselect`

node-6.11.2

I was getting this error upon running `npm run build`

```
ERROR in static/js/vendor.de7ff04081e219924cc1.js from UglifyJs
Unexpected token: operator (>) [./node_modules/vue-multiselect/src/multiselectMixin.js:23,0][static/js/vendor.de7ff04081e219924cc1.js:13877,32]
```

I found this [vue-select issue](https://github.com/sagalbot/vue-select/issues/71) which seemed to be a similar problem and added the following `resolve('node_modules/vue-multiselect')` to the include: in webpack.base.conf.js

```
     {
        test: /\.js$/,
        loader: 'babel-loader',
        include: [resolve('src'), resolve('test'), resolve('node_modules/vue-multiselect')]
      },
```

Now I'm getting this error:
```
"export 'default' (imported as 'Multiselect') was not found in 'vue-multiselect'
```

main.js
```
// Import Multiselect component
import Multiselect from 'vue-multiselect'
import 'vue-multiselect/dist/vue-multiselect.min.css'

Vue.component('multiselect', Multiselect)
```

Any idea what I'm doing wrong?  I'm thinking I have some version conflicts in my package.json?

**Comment by shentao** [comment]
Really weird.
I don’t think that adding that to the webpack config is the way to go so I would restore it to default. Try removing your node_modules directory or cache if on a CI tool. Also, please check whether your webpack and vue-loader plugins are up to date.
It might be a good idea to check if you can run `npm run build` on a different machine when using vue-multiselect.
Let me know if any of this helps.

**Comment by Namesonic** [comment]
Thanks for the feedback!  It does seem specific to my project configuration as I've used vue-multiselect in this project successfully prior to updating my packages.  I was using webpack 2.2.1 so I may have out of date conf's.

That webpack config change I made above didn't end up successfully loading the component anyways.  I restored to default and get the original error and yet the component builds and works in production anyways!

```
ERROR in static/js/vendor.cac4df3dad9997626b38.js from UglifyJs
Unexpected token: operator (>) [./node_modules/vue-multiselect/src/multiselectMixin.js:19,0][static/js/vendor.cac4df3dad9997626b38.js:13873,32]
```

I get the same error running `npm run build` on Windows 8 development box and on a CentOS 6.x production server.

I  had deleted node_modules and reinstalled during that test and did that again to make sure.  All packages reinstalled correctly, but no changes in the build error.

Using ncu I show all packages up to date, webpack seems to be up to date and my package.json as follows...

Interestingly, using an older webpack 2.2.1, I am able to compile this project successfully, see package.json as follows..

*New package.json*
```
{
  "name": "happ",
  "version": "1.0.0",
  "description": "A Vue.js project",
  "author": " <>",
  "private": true,
  "scripts": {
    "start": "npm run build",
    "dev": "node build/dev-server.js",
    "build": "node build/build.js",
    "lint": "eslint --ext .js,.vue src"
  },
  "dependencies": {
    "bootstrap": "^3.3.7",
    "bootstrap-vue": "^1.0.0-beta.7",
    "font-awesome": "^4.7.0",
    "vee-validate": "^2.0.0-rc.17",
    "vue": "^2.4.4",
    "vue-authenticate": "^1.2.7",
    "vue-multiselect": "^2.0.2",
    "vue-nl2br": "0.0.5",
    "vue-resource": "^1.3.4",
    "vue-router": "^2.7.0",
    "vuex": "^2.4.0"
  },
  "devDependencies": {
    "autoprefixer": "^7.1.4",
    "babel-core": "^6.26.0",
    "babel-eslint": "^8.0.0",
    "babel-loader": "^7.1.2",
    "babel-plugin-transform-runtime": "^6.23.0",
    "babel-preset-env": "^1.6.0",
    "babel-preset-es2015": "^6.24.1",
    "babel-preset-stage-2": "^6.24.1",
    "babel-register": "^6.26.0",
    "chalk": "^2.1.0",
    "chartist": "^0.11.0",
    "chartist-vuejs": "^0.9.2",
    "connect-history-api-fallback": "^1.3.0",
    "copy-webpack-plugin": "^4.0.1",
    "css-loader": "^0.28.7",
    "eslint": "^4.6.1",
    "eslint-config-standard": "^10.2.1",
    "eslint-friendly-formatter": "^3.0.0",
    "eslint-loader": "^1.9.0",
    "eslint-plugin-html": "^3.2.1",
    "eslint-plugin-import": "^2.7.0",
    "eslint-plugin-node": "^5.1.1",
    "eslint-plugin-promise": "^3.5.0",
    "eslint-plugin-standard": "^3.0.1",
    "eventsource-polyfill": "^0.9.6",
    "express": "^4.15.4",
    "extract-text-webpack-plugin": "^3.0.0",
    "file-loader": "^0.11.2",
    "friendly-errors-webpack-plugin": "^1.6.1",
    "function-bind": "^1.1.1",
    "html-webpack-plugin": "^2.30.1",
    "http-proxy-middleware": "^0.17.4",
    "json-loader": "^0.5.7",
    "node-dev": "^3.1.3",
    "node-sass": "^4.5.3",
    "opn": "^5.1.0",
    "optimize-css-assets-webpack-plugin": "^3.2.0",
    "ora": "^1.3.0",
    "parallelshell": "^3.0.1",
    "rimraf": "^2.6.2",
    "sass-loader": "^6.0.6",
    "semver": "^5.4.1",
    "style-loader": "^0.18.2",
    "url-loader": "^0.5.9",
    "vue-loader": "^13.0.4",
    "vue-strap": "^1.1.37",
    "vue-style-loader": "^3.0.3",
    "vue-template-compiler": "^2.4.4",
    "vuejs-datepicker": "^0.9.13",
    "webpack": "^3.5.6",
    "webpack-bundle-analyzer": "^2.9.0",
    "webpack-dev-middleware": "^1.12.0",
    "webpack-hot-middleware": "^2.19.1",
    "webpack-merge": "^4.1.0"
  },
  "engines": {
    "node": ">= 5.0.0",
    "npm": ">= 3.0.0"
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 8"
  ]
}
```

*Old package.json*

```
{
  "name": "happ",
  "version": "1.0.0",
  "description": "A Vue.js project",
  "author": " <>",
  "private": true,
  "scripts": {
    "start": "npm run build",
    "dev": "node build/dev-server.js",
    "build": "node build/build.js",
    "lint": "eslint --ext .js,.vue src"
  },
  "dependencies": {
    "bootstrap": "^3.3.7",
    "bootstrap-vue": "^1.0.0-beta.5",
    "font-awesome": "^4.7.0",
    "vee-validate": "^2.0.0-rc.14",
    "vue": "^2.2.2",
    "vue-authenticate": "^1.2.6",
    "vue-multiselect": "^2.0.2",
    "vue-nl2br": "0.0.5",
    "vue-resource": "^1.2.1",
    "vue-router": "^2.7.0",
    "vuex": "^2.2.1"
  },
  "devDependencies": {
    "autoprefixer": "^6.7.2",
    "babel-core": "^6.22.1",
    "babel-eslint": "^7.1.1",
    "babel-loader": "^6.2.10",
    "babel-plugin-transform-runtime": "^6.22.0",
    "babel-preset-env": "^1.2.1",
    "babel-preset-es2015": "^6.24.0",
    "babel-preset-stage-2": "^6.22.0",
    "babel-register": "^6.22.0",
    "chalk": "^1.1.3",
    "chartist": "^0.11.0",
    "chartist-vuejs": "^0.9.2",
    "connect-history-api-fallback": "^1.3.0",
    "copy-webpack-plugin": "^4.0.1",
    "css-loader": "^0.26.1",
    "eslint": "^3.14.1",
    "eslint-config-standard": "^6.2.1",
    "eslint-friendly-formatter": "^2.0.7",
    "eslint-loader": "^1.6.1",
    "eslint-plugin-html": "^2.0.0",
    "eslint-plugin-promise": "^3.4.0",
    "eslint-plugin-standard": "^2.0.1",
    "eventsource-polyfill": "^0.9.6",
    "express": "^4.14.1",
    "extract-text-webpack-plugin": "^2.0.0",
    "file-loader": "^0.10.0",
    "friendly-errors-webpack-plugin": "^1.1.3",
    "function-bind": "^1.1.0",
    "html-webpack-plugin": "^2.28.0",
    "http-proxy-middleware": "^0.17.3",
    "json-loader": "^0.5.4",
    "node-dev": "^3.1.3",
    "node-sass": "^4.5.2",
    "opn": "^4.0.2",
    "optimize-css-assets-webpack-plugin": "^1.3.0",
    "ora": "^1.1.0",
    "parallelshell": "^2.0.0",
    "rimraf": "^2.6.0",
    "sass-loader": "^6.0.3",
    "semver": "^5.3.0",
    "style-loader": "^0.18.2",
    "url-loader": "^0.5.7",
    "vue-loader": "^11.1.4",
    "vue-style-loader": "^2.0.0",
    "vue-template-compiler": "^2.2.4",
    "vuejs-datepicker": "^0.9.9",
    "webpack": "^2.2.1",
    "webpack-bundle-analyzer": "^2.2.1",
    "webpack-dev-middleware": "^1.10.0",
    "webpack-hot-middleware": "^2.16.1",
    "webpack-merge": "^2.6.1"
  },
  "engines": {
    "node": ">= 5.0.0",
    "npm": ">= 3.0.0"
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 8"
  ]
}
```

**Comment by shentao** [comment]
I think it might be related to this breaking change: https://github.com/vuejs/vue-loader/releases/tag/v13.0.0

**Comment by Namesonic** [comment]
I ended up replacing the vue-cli webpack template files in /build and /config with an updated copy from a fresh `vue-cli init webpack` installation and then updated all the package modules to their latest versions from the new package.json.

Thanks for the help on this and this is a great package!

**Comment by jbruni** [comment]
Context: fresh new `vue-cli init webpack` project, with dependencies updated by `ncu` and installed by `yarn`, using node version 6.12.0 and npm version 5.5.1.

While developing with `npm run dev` all went well.

When I tried to build with `npm run build`, I got:

> ERROR in my-project/js/vendor.4a615c4e87e2098b4355.js from UglifyJs
> Unexpected token: punc (() [./node_modules/vue-multiselect/src/pointerMixin.js:2,0][my-project/js/vendor.4a615c4e87e2098b4355.js:9733,7]

I tried the suggestion above: added `resolve('node_modules/vue-multiselect')` to babel-loader include configuration at _webpack.base.conf.js_.

And it worked!

>  Build complete.
